src/module/handlebars/BasicHelpers.ts

```diff
@@ -119,6 +119,10 @@ export const registerBasicHelpers = () => {
         return v1 + v2;
     });
 
+    Handlebars.registerHelper('or', function (v1, v2) {
```

Comment:
Have you tried the Foundry included helper?
![image](https://github.com/SR5-FoundryVTT/SR5-FoundryVTT/assets/146255/f27ce338-ed3a-4b54-8954-915b554c12ec)

Reply:
Hab meinen entfernt